arch/mips/Kconfig

```diff
@@ -11,6 +11,7 @@ choice
 config ARCH_CHIP_PIC32MX
 	bool "PIC32MX"
 	select ARCH_MIPS32
+  select ARCH_HAVE_EIC
```

Comment:
Need to use TABs in Kconfig files, not 4 spaces.  There are several places that need to be fixed.